### DISCUSSION TITLE
Create a new project from console

### DISCUSSION THREAD

**Discussion by Epado** [mia-platform/community#192] [Q&A] [answered]
Hi everyone,

When creating a new project from the console, the second step asks you to pick a template for the project itself, among the choices there is:
- Empty 
- Mia-Platform basic project template

Can you tell me the differences ? How can i know what a specific template will offer ?

**Comment by davidebianchi** [comment, marked as answer]
Hi @Epado! 
Sure, we should improve the description of what a template contains. This answer refers on the template present on the SaaS.

* The empty template it's an empty template. So, the project which will generate is without configuration (there aren't default deploy pipelines, default kustomization configuration, gitignore and so on).
* Mia-Platform basic project template does not have pre configured microservices (which could be easily configured from the marketplace) but contains a default environment and kustomization configuration, with the Traefik default ingress route configured (needed to expose the project with a public hostname), contains the GitLab pipeline and the mlp configuration files.
